net_udp.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 The Prometheus Authors
+// Copyright 2020 The Prometheus Authors
```

Comment:
@pgier We don't update copyright dates in Prometheus repos. Only the initial date is required. We don't need to revert this, but we should avoid updating these in the future.

Reply:
Ah sorry, didn't notice that change when I reviewed this.  I agree, make sense not to change these in the future.